packages/gen-ai-hub/src/client/openai/openai-client.test.ts

```diff
@@ -62,7 +62,11 @@ describe('openai client', () => {
       const response = await client.chatCompletion(prompt, {
         deploymentId: '1234'
       });
-      expect(response).toEqual(mockResponse);
+
+      expect(response).toBeInstanceOf(OpenAiChatCompletionResponse);
+      expect(response.data).toEqual(mockResponse);
+      expect(response.getContent()).toEqual(expect.any(String));
+      expect(response.getFinishReason()).toEqual(expect.any(String));
```

Comment:
In the orchestration test you also test the .getUsageTokens() function, is this intentionally left out?